src/Grpc.AspNetCore.Server/Internal/Constants.cs

```diff
@@ -0,0 +1,29 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using Grpc.Core;
+using Microsoft.Extensions.Primitives;
+
+namespace Grpc.AspNetCore.Server.Internal
+{
+    internal static class Constants
+    {
+        public const string GrpcStatusHeader = "grpc-status";
+        public static readonly StringValues GrpcStatusOk = ((int)StatusCode.OK).ToString();
```

Comment:
this one got me confused - I first though it evaluates to the string value "OK" (which would be against https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md), but then realized it's `"0"`. 
Perhaps making it obvious that these are numbers in string form would be good.
Fine to leave as is, but let's come up with something more readable once we start supporting non-ok statuses as well.

Reply:
Lets change it to `public const string GrpcStatusOk = "0";`

Reply:
We lose the reference to StatusCode.OK but I guess we can do that in a comment.